.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: python
 python:
   - "3.5"
+before_install:
+  - sudo apt-get -y install unixodbc-dev
```

Comment:
So we think this should now be unecessary if you bump `opensafely-cohort-extractor`, right?

Reply:
That'd be nice!